### ISSUE TITLE
Input only the desired scene 

### ISSUE THREAD

**Issue by GUE1RNG** [traveller59/second.pytorch#497]
Hello

I am working with StreamPETR and dataset is NuScenes, I want to pass only one scene rather than passing all the scenes in validation set!

Any insights would be good.
